jekyll/_ccie/aws.md

```diff
@@ -1,69 +1,76 @@
 
 ## Prerequisites
 
-* AWS Access Key
-* AWS Secret Key
-* AWS Region (e.g. "us-west-2")
-* AWS VPC ID
-	* Ensure that your VPC has the setting [enableDNSSupport](https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/vpc-dns.html#vpc-dns-updating)
-	set to true
-	* Instructions if you have a [Private Subnet]({{site.baseurl}}/enterprise/aws-private-subnet/)
-* AWS Subnet ID
-* CircleCI License file (.rli)
+Install the following automated infrastructure provisioning software:
+
+* Terraform, see the [Download Terraform](https://www.terraform.io/downloads.html) web site for links to packages for your architecture. To install CircleCI without using Terraform, refer to the [Manual AWS Installation]({{site.baseurl}}/enterprise/aws-manual/).
+
+Have the following information available before beginning the installation procedure:
+
+* CircleCI License file (.rli), contact [CircleCI Enterprise support](mailto:enterprise-support@circleci.com) for a license.
+* AWS Access Key, AWS Secret Key, and AWS Subnet ID
+* AWS Region, for example `us-west-2`
+* AWS Virtual Private Cloud (VPC) ID. Your default VPC ID is listed under Account Attributes in Amazon if your account is configured to use a default VPC.
+* Set your VPC [`enableDnsSupport`] setting to `true` to ensure that queries to the Amazon provided DNS server at the 169.254.169.253 IP address, or the reserved IP address at the base of the VPC IPv4 network range plus two will succeed. See the [Using DNS with Your VPC](https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/vpc-dns.html#vpc-dns-updating) Amazon Web Servies documentation for additional details.
+	
+## Private Subnet Requirements	
 
-## Setup Terraform
+The following additional settings are required to support using private subnets on AWS with CircleCI:
 
-(If you prefer to do this process manually or with different tools please see the [Manual AWS Instructions]({{site.baseurl}}/enterprise/aws-manual/).)
+- The private subnet for builder boxes must be a network address translation (NAT) instance or an internet gateway configured for the outbound traffic to the internet.
+- The [VPC Endpoint for S3](https://aws.amazon.com/blogs/aws/new-vpc-endpoint-for-amazon-s3/) should be enabled. Enabling the VPC endpoint for S3 should significantly improve S3 operations for CircleCI and other nodes within your subnet.
+- Adequately power the NAT instance for heavy network operations.  Depending on the specifics of your deployment, it is possible for NAT instances to become constrained by highly parallel builds using Docker and external network resources.  A NAT that is inadequate could cause slowness in network and cache operations.
+  - If you are integrating with https://github.com/, ensure that your network access control list (ACL) whitelists github.com webhooks.  When integrating with GitHub, either set up CircleCI in a public subnet, or set up a public load balancer to forward github.com traffic.
```

Comment:
I think the ACL needs to be more open so the services/build can download build images, but I'd double-check with someone else.